hypothesis.py

```diff
@@ -89,6 +88,7 @@ def fraud_metrics(sample_data: pd.DataFrame,
         _LONGITUDE_COLUMN_NAME,
         _LATITUDE_COLUMN_NAME,
         _FRAUDULENT_COLUMN_NAME])[isotope_column_name]
+      
```

Comment:
Could we revert the changes on this file? It looks like we're only adding line breaks that we don't need for this change.